app/assets/stylesheets/widgets/form.scss

```diff
@@ -11,90 +15,95 @@
       font-size: 11px;
   }
 
+  label {
+    display: block;
+    margin: 0px;
+  }
+
   .inputs {
-    display: table;
+    display: table-row;
     width: 100%;
-    border-spacing: 5px;
 
-    .label {
+    .input {
       display: table-cell;
       padding: 5px;
     }
-  }
 
-  label, .input {
-    display: block;
-    margin: 0px;
-  }
-
-  .input > span {
+    .input > span {
       width: 20%;
       padding-right: 10px;
       margin-top: 10px;
       color: #888;
-  }
+    }
 
-  .chosen-container, textarea, select,
-  input[type="search"], input[type="text"], input[type="number"], input[type="password"], input[type="email"],
-  input[type="checkbox"], input[type="radio"], input[type="tel"] {
-    border: 1px solid #DADADA;
-    color: #888;
-    height: 30px;
-    margin-bottom: 16px;
-    margin-right: 9px;
-    margin-top: 2px;
-    outline: 0 none;
-    padding: 3px 3px 3px 5px;
-    width: 100%;
-    font-size: 12px;
-    line-height:15px;
-    box-shadow: inset 0px 1px 4px #ECECEC;
-    -moz-box-shadow: inset 0px 1px 4px #ECECEC;
-    -webkit-box-shadow: inset 0px 1px 4px #ECECEC;
-  }
+    .chosen-container, textarea, select,
+    input[type="search"], input[type="text"], input[type="number"], input[type="password"], input[type="email"],
+    input[type="checkbox"], input[type="radio"], input[type="tel"] {
+      border: 1px solid #DADADA;
+      color: #888;
+      height: 30px;
+      margin-bottom: 16px;
+      margin-right: 9px;
+      margin-top: 2px;
+      outline: 0 none;
+      padding: 3px 3px 3px 5px;
+      width: 100%;
+      font-size: 12px;
+      line-height:15px;
+      box-shadow: inset 0px 1px 4px #ECECEC;
+      -moz-box-shadow: inset 0px 1px 4px #ECECEC;
+      -webkit-box-shadow: inset 0px 1px 4px #ECECEC;
+    }
 
-  input[type="checkbox"], input[type="radio"] {
-    width: inherit;
-    height: inherit;
-  }
+    input[type="checkbox"], input[type="radio"] {
+      width: inherit;
+      height: inherit;
+    }
 
-  textarea{
-    padding: 5px 3px 3px 5px;
-    height:100px;
-  }
+    textarea{
+      padding: 5px 3px 3px 5px;
+      height:100px;
+    }
 
-  select {
-    background: #FFF url('down-arrow.png') no-repeat right;
-    appearance:none;
-    -webkit-appearance:none;
-    -moz-appearance: none;
-    text-indent: 0.01px;
-    text-overflow: '';
-    width: 70%;
-    height: 35px;
-    line-height: 25px;
-  }
+    select {
+      background: #FFF url('down-arrow.png') no-repeat right;
+      appearance:none;
+      -webkit-appearance:none;
+      -moz-appearance: none;
+      text-indent: 0.01px;
+      text-overflow: '';
+      width: 70%;
+      height: 35px;
+      line-height: 25px;
+    }
 
-  .nested-fields {
-    margin-left: 10%;
-    margin-bottom: 16px;
-    margin-top: 2px;
-  }
+    .nested {
+      display: table;
+      width: 96%;
+      margin-left: 5%;
+      .nested-fields {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4